src/Datadog.Trace/Tracer.cs

```diff
@@ -586,7 +586,7 @@ internal async Task WriteDiagnosticLog()
                     writer.WriteEndObject();
                 }
 
-                Log.Information("DATADOG TRACER CONFIGURATION - {0}", stringWriter.ToString());
+                Log.Debug("DATADOG TRACER CONFIGURATION - {Configuration}", stringWriter.ToString());
```

Comment:
Are we okay with only receiving this kind information after a user intentionally turns debug on?
Won't this hurt support?

Reply:
You’re right this should be sent as Information.

Reply:
I just posted my review and I can see we will have many duplicate comments 😄 
Yes, we always want to emit this line. It is standard across all language tracers. The issue is that the RFC only defines `Debug`, Warning`, and `Error` levels, so we should probably update the RFC.

Reply:
Reverted this change back to Information